lessons/Data-Science/ai-linear-regression/langs/en-us/index.adoc

```diff
@@ -0,0 +1,127 @@
+= Artificial Intelligence &  Regression
+
+@description{Students explore regression and supervised machine learning, exploring the functioning of ALVINN (the world's first self-driving car) as motivation.}
+
+@ifproglang{pyret}{
+@lesson-prereqs{ds-intro}
+}
+
+@ifproglang{codap}{
+@lesson-prereqs{ds-intro}
+}
+
+@keywords{}
+
+[@lesson-intro-table]
+|===
+| Lesson Goals
+| Students will be able to...
+
+- define supervised machine learning
+- describe how regression is one form of supervised learning
+- describe the function that automated the steering of the very first self-driving car
+
+| Student-facing Lesson Goals
+|
+
+- Let's learn about the functioning of the very first self-driving car!
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+|===
+
+== Regression and Supervised Learning @duration{10 minutes}
+
+=== Overview
+Students learn about ALVINN (autonomous Land Vehicle in a Neural Network) as an example of supervised machine learning and regression.
+
+=== Launch
+
+@right{@image{images/navlab.png, 400}}
+
+Meet ALVINN (pictured to the right).
+
+ALVINN (short for Autonomous Land Vehicle in a Neural Network) was developed in 1989 by a team of computer scientists at Carnegie Mellon University. ALVINN is the forebear of today's self-driving cars: a retrofitted Army ambulance equipped with sensors, computers, and actuators (the part of a machine that allows movement). ALVINN had about one-tenth of the processing power of an Apple Watch - but was able to safely drive at 70 miles per hour (albeit on a very closed course) by the early 1990s! How did it happen?
+
+As with all machine learning, ALVINN required training. The goal of that training? Learning how to steer. Here's what ALVINN's training entailed:
+
+- A person drives the vehicle (known as the Navlab - short for navigation lab) as ALVINN watches the road through a front-facing video camera.
+- Every two seconds, ALVINN digitizes an image of the road ahead and records the chosen steering direction.
+- Training images are reduced in resolution to 30x32 pixels and provided as inputs. The corresponding outputs are the "correct" steering angles.
+
+
+[cols="^.^1,^.^1", stripes="none", options="header"]
+|===
+
+| x (input data / feature) | y (target data / output)
```

Comment:
give these images more room! they're so _tiny!_

Reply:
okay!

I hate having empty unused white space in the lesson. But i can get over it.